src/snyk/analytics/itly.ts

```diff
@@ -149,6 +151,8 @@ export class Iteratively {
         },
       },
     );
+
+    return itly.flush();
```

Comment:
@michelkaporin, question: why we need to call `flush` after pluginIsInstalled?

Reply:
@pavel-snyk to prevent out of order event delivery that happens between 'Plugin is Installed' and 'Welcome is Viewed' events currently. They are fired one after another and Iteratively/Segment ingest them out of order for some reason.